deeppavlov/models/preprocessors/sentseg_preprocessor.py

```diff
@@ -0,0 +1,26 @@
+from nltk import word_tokenize
+from overrides import overrides
+from typing import List
+
+from deeppavlov.core.common.registry import register
+
+
+@register("sentseg_restore_sent")
+def SentSegRestoreSent(batch_x: List[List[str]], batch_y: List[List[str]]) -> List[str]:
+    ret = []
+    for x, y in zip(batch_x, batch_y):
```

Comment:
I feel like `x` is tokens and `y` is some kind of tags? Can there be more meaningful variable names? And maybe a docstring if variable names are not enough.

Reply:
changed x -> words, y -> tags